### PR TITLE
Upgrade Test Suite to Quarkus Test Framework 0.0.4

### PR DESCRIPTION
**Pull request by Sgitario**
Quarkus Test Framework 0.0.4 has not been released yet.

Fix https://github.com/quarkus-qe/quarkus-test-suite/issues/112

### CODE REVIEW

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
run tests

File: properties/src/main/java/io/quarkus/qe/properties/providers/CustomConfigSource.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class CustomConfigSource implements ConfigSource {
 
     private static final int ORDINAL = 999;
-    private static final String PROPERTIES_FILE = "/configsource.properties";
+    private static final String PROPERTIES_FILE = "/configsource.prop";
```
Comment (Sgitario) [inline properties/src/main/java/io/quarkus/qe/properties/providers/CustomConfigSource.java:15]: This is caused by https://github.com/quarkus-qe/quarkus-test-framework/issues/169

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
@rsvoboda besides the OCP failures, I think we should handle those at a later stage (when releasing 0.0.5 or as a separate task).
If you agree, this PR is ready to be merged.

**Review by pjgg — APPROVED** [review]
Lets wait to CI jobs

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**